libraries/DataFlash/LogStructure.h

```diff
@@ -1549,6 +1573,11 @@ enum LogMessages : uint8_t {
     LOG_ISBD_MSG,
     LOG_ASP2_MSG,
     LOG_PERFORMANCE_MSG,
+    LOG_MAV0_MSG,
+    LOG_MAV1_MSG,
+    LOG_MAV2_MSG,
+    LOG_MAV3_MSG,
+    LOG_MAV4_MSG,
```

Comment:
Last dev call we had an interesting discussion about having multiple messages or one message with ID - same issue I mentioned in #7831.
I don't know what the solution is (at least, without DF parsing programs allowing to split messages by one field), but if we keep adding multiples of the same message we are going to exhaust message IDs fast.

Reply:
This keeps coming up, the multiple messages is unsustainable, and we need to solve the tooling side on it and move on...

Reply:
Moving to having an instance field.